COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
> Because according my terms of employment say that they own the copyright on anything I produce while I work for them.
Can you do that at home at your free time? :)